speech/recognize.js

```diff
@@ -180,13 +192,184 @@ function syncRecognizeWords (filename, encoding, sampleRateHertz, languageCode)
   // [END speech_sync_recognize_words]
 }
 
+function syncRecognizePunctuation (filename, encoding, sampleRateHertz, languageCode) {
+  // [START sync_recognize_punctuation]
+  // Imports the Google Cloud client library
+  const fs = require('fs');
+  const Speech = require('@google-cloud/speech');
+
+  // Creates a client
+  const speech = Speech.v1beta1();
```

Comment:
Might be of a different version - let's wait until the client library becomes available then confirm/update this.